tests/test_bs.py

```diff
@@ -415,9 +414,13 @@ def test_transaction(set_namespace, setup_clients):
     assert 0 == nonce_val
 
     api = 'v1/submittransaction'
-    data = '{"sender":"1","reciever":"222","nonce":"0","amount":"100"}'
+    data = '{"srcAddress":"1","dstAddress":"222","nonce":"0","amount":"100"}'
 
     out = api_call(client_ip, data, api, testconfig['namespace'])
     assert '{"value":"ok"}' in out.decode("utf-8")
+    time.sleep(60 * 10)
```

Comment:
10 minutes 😮 ?

Reply:
lets start there and see how low can we go
#822